docs: add pack cve handling policy DOC-2984 (#11048)
spectrocloud/librarium: docs/docs-content/integrations/cve-policy.md

| --------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| Fix available upstream, pack is on the latest version     | The fix is already incorporated; no further action is needed.                                                                                                 |
| Fix available upstream, pack is behind the latest version | We update the pack to the upstream version that contains the fix.                                                                                             |
| No fix available upstream                                 | We update the pack to the latest available version. We also track the CVE and remediate it once the upstream project ships a fix and we pick that version up. |

#### Upstream Version Tracking

We continuously track upstream releases and target the latest available upstream version of each pack in our regular
release cadence. This is the primary way fixes reach you. When an upstream project patches a vulnerability, we pick up
the patched version promptly.

> Some upstream projects release frequently, sometimes multiple times per week. In those cases, a pack may briefly trail

Reviewer: ```suggestion
Some upstream projects release frequently, sometimes more than once per week. In those cases, a pack may briefly trail
```

Reviewer: Just read slightly oddly.

#### Why Do Some Packs Show High CVE Counts?

A pack contains multiple images, each with an OS layer, runtimes, and the application. Counts aggregate findings across
all of these. Many are base OS or library findings without an available fix, or are non-exploitable in context. We focus
on CVEs that have an available upstream fix.

#### Why Does a Pack Still Report a CVE That Has a Published Fix?

A fix being available in the vulnerability database does not mean the upstream project has already incorporated it into
its release. We treat such CVEs as actionable and update the pack once an upstream version that includes the fix is
available.

#### Are Packs Always on the Latest Upstream Version?

We target the latest upstream version for each pack on our regular cadence. High-frequency upstream projects may briefly
trail the newest release, and lower-priority packs may occasionally lag by a version or two. In all cases, we aim to
incorporate available fixes promptly.

#### Where Can I Find the CVEs for a Specific Pack?

Per-pack CVE data is not published in the Spectro Cloud documentation. For questions about a specific pack, contact

> Spectro Cloud support. You can also scan the pack yourself using scanners such as `Trivy` or `Grype`.

Reviewer: ```suggestion
Spectro Cloud support. You can also scan the pack yourself using open source vulnerability scanners such as `Trivy` or `Grype`.
```

Reviewer: Maybe?
